content/faq.md

```diff
@@ -40,48 +42,41 @@ title = "Frequently Asked Questions (FAQ)"
 
 * When will it be possible to play online in multiplayer?
 
-> It is already possible to play multiplayer, although there are some bugs in this regard. For example there is unfixed yet bug about random map generator deynchronization when playing VCMI across multiple platforms(windows/linux), so you may see wood on adventure map but when you pick it up you collect ore etc. There are also some crashes to fix regarding player disconnection.
+> As of VCMI 1.1.0 multiplayer reached fair level of stability and is playable. There is multiplayer lobby that makes it way easier to setup multiplayer session, but as of VCMI 1.1.0 it unfortunately isn't available on all platforms. We know that multiplayer lobby on Android is potentially one of top wanted features, but also hard to get done.
 
 * How close is VCMI emulating H3 gameplay?
 
-> We try to make VCMI as close to Heroes 3 as possible and we want to deliver 100% same game mechanics, with original heroes 3 bugs such as infinite ammo for ballista becoming toggle on/off switches. AI will not be the same, we deliver our custom adventure map and battle AI. Currently there are not many game mechanics bugs that will be noticed by people who do not have advanced knowledge about the game. Some examples are: Lack of moat in tower type town, some spell mechanics, spell hero specialties are bugged. There are  various display glitches, but they have less priority than other things.
-
-* I updated VCMI Essential Files in VCMI launcher and I have some bugs.
+> We try to make VCMI as close to Heroes 3 as possible and we want to deliver 100% same game mechanics, with some original heroes 3 bugs such as infinite ammo for ballista potentially becoming toggle on/off switches, if proper fix is undesired by players. AI will not be the same, we deliver our custom adventure map and battle AI. Currently as of VCMI 1.1.0 there are not too many game mechanics bugs that will be noticed by people who do not have advanced knowledge about the game. Check our (issues list)[https://github.com/vcmi/vcmi/issues] to get some examples about currently reported bugs & missing features. There are also various display glitches, they are also subjects to get fixed. Another notable t hing is lack of various HD Mod features, which brings lots of QoL changes to original game. We also plan to support its stuff as integral part of VCMI, with on/off switch for features that alter original game elements.
```

Comment:
Simplify:

VCMI follows original Heroes 3 mechanics as close as possible and should be fully compatible with it. However, some obious bugs were fixed, while some features were made optional. The AI is brand new and completely different.